torch/_inductor/lowering.py

```diff
@@ -1730,10 +1730,6 @@ def apply_constraint(arg, fx_arg):
 # ROCm specific fallback, perf issues are observed when registered
 make_fallback(aten.miopen_batch_norm, warn=False)
 
-if torch.version.hip is not None and torch.cuda.is_available():
-    # tl.reduce not available yet in ROCm's version of triton
-    make_fallback(aten.prod, warn=False)
```

Comment:
Can you also remove everything that references `TRITON_HAS_REDUCE` in `triton_helpers.py`